uptane/clients/secondary.py

```diff
@@ -483,20 +502,34 @@ def fully_validate_metadata(self):
 
     # Refresh the top-level metadata first (all repositories).
     self.updater.refresh()
-
     validated_targets_for_this_ecu = []
 
     # Comb through the Director's direct instructions, picking out only the
     # target(s) earmarked for this ECU (by ECU Serial)
     for target in self.updater.targets_of_role(
         rolename='targets', repo_name=self.director_repo_name):
 
-      # Ignore target info not marked as being for this ECU.
```

Comment:
My bad, I must be changing some part of the code and might have deleted the comment by mistake.